CHANGELOG.md

```diff
@@ -644,3 +644,17 @@ Other minor additions
                           (isIdempotentCommutativeMonoid : IsIdempotentCommutativeMonoid ∙ ε)
 
   ```
+
+* Added a new module `Relation.Nullary.Irrelvant`:
+  ```agda
+  Irrelevant : ∀ {p} → Set p → Set p
+  Irrelevant P = ∀ (p₁ p₂ : P) → p₁ ≡ p₂
+  ```
```

Comment:
There is a typo in `Irrelevant` in the module name.
Plus you can drop the type annotation in the CHANGELOG.